include/CMakeLists.txt

```diff
@@ -8,3 +8,18 @@ file(GLOB_RECURSE perf_headers perf/*.h perf/*.hpp)
 set(PERF_HEADERS "${perf_headers}" PARENT_SCOPE)
 
 add_subdirectory(read)
+
+file(MAKE_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build")
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/opencv" -B "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build" -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+)
+execute_process(
+    COMMAND ${CMAKE_COMMAND} --build "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build" --config "${CMAKE_BUILD_TYPE}"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+)
```

Comment:
Why do you build OpenCV one more time?

Reply:
Because the person who worked before connected reading only for the "read" folder and I did not see OpenCV in the folder from which I worked. So I rewrote include for everything. And please see what's wrong with the bot. Maybe I'm doing something wrong and I don't understand, but it seems like I took the changes from the main, but the situation is the same.

Reply:
Need to move OpenCV build to 3rdparty. Cause here we are duplicating OpenCV build. As for the bot, consider this as a non-blocking issue, if we don't find the root cause until all comments resolved and 3 approvals collected, we'll merge it in any case

Reply:
fixed